e2e/tests/room/09-jumptomessage.spec.ts

```diff
@@ -41,9 +41,6 @@ async function clearCache() {
 	await waitFor(element(by.id('rooms-list-view')))
 		.toBeVisible()
 		.withTimeout(5000);
-	await waitFor(element(by.id('rooms-list-view-item-jumping')))
```

Comment:
Why are you just removing an assertion?
It's clear we still need it to be sure it's there.
Am I missing something?